docker-compose.yml

```diff
@@ -52,6 +52,7 @@ services:
       - AIRFLOW__KUBERNETES__IN_CLUSTER=False
       - URL=http://localhost:8000
       - WEBSERVER_USE_RBAC=False
+      - 'AIRFLOW_CONN_ADM_SFTP=ftp://myname:mypassword@myhost.com:8000?known_hosts=myhost.com+AAAABBBBB'
```

Comment:
I'm adding dummy connection details for the "adm_sftp" connection as env var here to allow the dag parsing test to run successfully. This appears to be working, and may be quicker to load than using the CLI to add the connection in `bin/run`. AFAICT, there's overhead to every airflow CLI call that's dominating the startup time.

Reply:
This was an option i considered as well. It becomes a little more complex with gcp/aws type connections, and we can use a .env file to store all of them as well. 

https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#storing-a-connection-in-environment-variables

Why is the whole k:v in single quotes? Shouldn't it be k='v'?

Reply:
From the yaml document's perspective, this is a list of strings. It's docker-compose that is expecting to parse these items as `k=v`. If we wrote `k='v'` here, then the value would literally contain the quotes.

That said, I don't think there are any characters here that are problematic, so we don't need any quotes at all.